src/components/re-entry/ReentryButtons.tsx

```diff
@@ -0,0 +1,41 @@
+'use client';
+import Link from 'next/link';
+import { usePathname } from 'next/navigation';
+import { useTranslations } from 'next-intl';
+
+import { generatePdf } from '@/libs/Pdf/Pdf';
+import Button from '@/ui/button/button';
+
+type ReentryButtonsProps = {
+  title: string;
+};
+
+const ReentryButtons = ({ title }: ReentryButtonsProps) => {
+  const tCommon = useTranslations('Common');
+
+  const pathname = usePathname();
+
+  const handleDownloadPdf = () => {
+    generatePdf(title, pathname);
+  };
+
+  return (
+    <div className="govuk-button-group">
```

Comment:
Same like for conjunction use ButtonsGroup

Reply:
Fixed